template/androidApp/src/main/java/io/bloco/template/navigation/NavHost.kt

```diff
@@ -0,0 +1,26 @@
+package io.bloco.template.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import io.bloco.template.features.details.DetailsScreen
+import io.bloco.template.features.list.ListScreen
+
+@Composable
+fun TemplateNaveHost(
```

Comment:
Why TemplateNaveHost instead of TemplateNavHost? (Its NavController, NavHost..)

Reply:
ups :D